### PR TITLE
TVL and TVS from xexchange.com

### PR DESCRIPTION
**Pull request by dragos-rebegea**
**NOTE**

> If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).

1. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
2. The protocol is usually listed within 24 hours of merging the PR
3. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
4. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
5. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---

### CODE REVIEW

**Comment by llamatester** [comment]
Error while running adapter at projects/maiar: 
        
 
  
    
    
    Cannot query field "totalValueStakedUSD" on type "FactoryModel". Did you mean "totalValueLockedUSD"?
    Truncated error stack:
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/graphql-request/dist/index.js:359:31

**Comment by llamatester** [comment]
The adapter at projects/maiar exports TVL: 
        
 
 
    elrond                    102.54 M
    elrond-staking            164.97 M
    staking                   164.97 M
    
    total                    102.54 M

File: projects/maiar/index.js [hunk 1]
```diff
@@ -1,32 +1,35 @@
...
-      pairCount
-      totalValueLockedUSD
-    }
+    totalValueLockedUSD
+  }
+`
+
+const StakeQuery = gql`
+{
+    totalValueStakedUSD
```
Comment (g1nt0ki) [inline projects/maiar/index.js:12]: hi @dragos-rebegea how is this computed? according to coingecko, marketcap of MEX is 81M, but this is twice that

**Comment by g1nt0ki** [comment]
@dragos-rebegea sorry, this PR was correct, I didnt understand it included metastaking, I ended up with similar code in the end: https://github.com/DefiLlama/DefiLlama-Adapters/commit/8294f31231b21ccf5033f5a99e800821e81297e8

**Pull request closed without merging**